spec/dummy/config/database.yml

```diff
@@ -0,0 +1,3 @@
+test:
+  adapter: sqlite3
```

Comment:
I think with JRuby you will need to conditionally add https://rubygems.org/gems/jdbc-sqlite3 from the Gemfile

Reply:
I'll try to figure out how to do that

Reply:
In the Gemfile (since that should have development dependencies), we can do
```
if defined?(JRUBY_VERSION)
  gem 'jdbc-sqlite3'
else
  gem 'sqlite3'
end
```

or something like this.